apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes/ast.ex

```diff
@@ -64,4 +64,77 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST d
   defp do_to_pipe(node, acc) do
     {node, acc}
   end
+
+  def ast_to_string(ast) do
```

Comment:
This is an attempt to handle escape characters as expected. Local tests passes successfully, but I'm not sure if this is the best way to approach the problem